python/ray/data/_internal/lazy_block_list.py

```diff
@@ -502,7 +504,9 @@ def _submit_task(
         self, task_idx: int
     ) -> Tuple[ObjectRef[MaybeBlockPartition], ObjectRef[BlockPartitionMetadata]]:
         """Submit the task with index task_idx."""
-        stats_actor = _get_or_create_stats_actor()
+        if self._stats_actor is None:
```

Comment:
do we need the null check since it is already using get_if_exists=True

Reply:
This is actually the cached actor handle in this class which was initialized to None, so it's None for the first call here.